templates/users/followers.html.tera

```diff
@@ -1,13 +1,19 @@
 {% extends "base" %}
 
 {% block title %}
-{{ user.display_name }}'s Followers
+{% if user.display_name %}
```

Comment:
Same here: it would be great if you could move this `if` outside of the `title` block.